src/lists.py

```diff
@@ -8,22 +8,34 @@
 
 # Change x so that it is [1, 2, 3, 4]
 # YOUR CODE HERE 
+x.append(4)
 print(x)
 
 # Using y, change x so that it is [1, 2, 3, 4, 8, 9, 10]
 # YOUR CODE HERE 
+x.extend(y)
 print(x)
 
 # Change x so that it is [1, 2, 3, 4, 9, 10]
 # YOUR CODE HERE 
+x.remove(8)
 print(x)
 
 # Change x so that it is [1, 2, 3, 4, 9, 99, 10]
-# YOUR CODE HERE 
+# YOUR CODE HERE
+def insertnum():
+    for num in x:
+        if num == 9:
+            # print(x.index(9) + 1)
+            x.insert(x.index(9) + 1, 99)
+insertnum()
 print(x)
 
 # Print the length of list x
 # YOUR CODE HERE 
-
+print("length of x: " + str(len(x)))
 # Print all the values in x multiplied by 1000
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+multiple = map(lambda z: z*1000, x )
```

Comment:
Nice use of a map and a lambda function. I've only ever used lambdas a few times, but this looks correct.